rules/defense_evasion/clear_kernel_ring_buffer.yaml

```diff
@@ -12,4 +12,5 @@
   severity: low
   description: Monitors for the execution of the dmesg utility with the clear flag. Attackers may clear kernel ring buffer events to 
     evade detection after installing a Linux kernel module (LKM).
-  condition: header.image == "/usr/bin/dmesg" AND payload.argv CONTAINS "-c"
\ No newline at end of file
+  condition: (payload.filename STARTS_WITH "/usr/bin/dmesg" OR payload.filename STARTS_WITH "/bin/dmesg" OR payload.filename STARTS_WITH "/usr/sbin/dmesg")
+    AND (payload.argv CONTAINS "-c" OR payload.argv CONTAINS "-C" OR payload.argv CONTAINS "--clear")
```

Comment:
- `--read-clear` missing
- keep only `payload.filename == "/usr/bin/dmesg"`

Reply:
check if possible to use fileopen event and flags O_TRUNC and/or O_WRITE, O_RDWR